pkg/config/log.go

```diff
@@ -129,7 +129,7 @@ func SetupLogger(logLevel, logFile, uri string, rfc, logToConsole, jsonFormat bo
 type ErrorLogWriter struct{}
 
 func (s *ErrorLogWriter) Write(p []byte) (n int, err error) {
-	seelog.Error(string(p))
+	log.ErrorStackDepth(6, fmt.Sprintf("Error from the agent http API server: %s", strings.TrimSpace(string(p))))
```

Comment:
the `ErrorLogWriter` isn't meant to be specific to the HTTP API server, so the string prefix "Error from the agent [...]" shouldn't be added here. Maybe setting it as the prefix of the golang `Logger`  that we pass to the server would make more sense

Reply:
also, in the general use case of `ErrorLogWriter`, I don't think we can expect a user of `ErrorLogWriter` to always want a depth of `6`, so I'd make this configurable on the writer